custom-namespaces.yaml

```diff
@@ -1960,3 +1960,26 @@ jira_service_desk:
       type: table_explore
       views:
         base_view: user
+zoom:
+  pretty_name: Zoom
+  connection: bigquery-oauth
+  owners:
+    - rantwan@mozilla.com
+  glean_app: false
+  views:
+    meetings:
+      type: table_view
```

Comment:
Does it make sense to also generate `table_explore`s for these?

Reply:
My plan is to have one main "meetings" explore joining several of these views together, and a "users" explore just on the `users` view.  So while I could add a `table_explore` for the latter here, I figured since I'll be needing to do a looker-spoke-default PR for the other explore I'd just do both that way.